torch/_inductor/codegen/wrapper.py

```diff
@@ -1530,12 +1530,16 @@ def prepare_triton_kernel_call(self, device_index, call_args):
         def wrap_arg(arg):
             if isinstance(arg, str):
                 # dynamo wraps unspec variable as 0d CPU tensor, need convert to scalar
-                return arg + ".item()" if V.graph.is_unspec_arg(arg) else arg
+                if V.graph.is_unspec_arg(arg) or arg in zero_dim_cpu_tensor_list:
+                    return f"{arg}.item()"
+                else:
+                    return arg
             elif isinstance(arg, (int, float, bool, SymbolicCallArg)):
                 return str(arg)
             else:
                 return self.expr_printer(V.graph.sizevars.simplify(arg))
 
+        zero_dim_cpu_tensor_list = V.graph.check_0dim_cpu_tensor(call_args)
```

Comment:
Is it possible to get the kernel's zero_dim_cpu_tensor_list rather than generating it again?

Reply:
move it to graph.py